### PR TITLE
Added Whatsapp & crypto APIs

### PR DESCRIPTION
**Pull request by fayazara**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] Your additions are ordered alphabetically
- [x] Your submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column should be padded with one space on either side
- [x] You have searched the repository for any relevant issues or pull requests
- [x] Any category you are creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,6 +157,7 @@ API | Description | Auth | HTTPS | CORS | Link |
 API | Description | Auth | HTTPS | CORS | Link |
 |---|---|---|---|---|---|
 | Binance | Exchange for Trading Cryptocurrencies based in China | `apiKey` | Yes | Unknown | [Go!](https://github.com/binance-exchange/binance-official-api-docs) |
+| BitBNS | Indian Cryptocurrency exchange price ticker | No | Yes | No | [Go!](https://bitbns.com/order/getTickerWithVolume/) |
```
Comment (davemachado) [inline README.md]: Please link to documentation rather than a live endpoint.

File: README.md [hunk 2]
```diff
@@ -626,6 +628,7 @@ API | Description | Auth | HTTPS | CORS | Link |
 | Twitch | Game Streaming API | `OAuth` | Yes | Unknown | [Go!](https://dev.twitch.tv/docs) |
 | Twitter | Read and write Twitter data | `OAuth` | Yes | Unknown | [Go!](https://dev.twitter.com/rest/public) |
 | vk | Read and write vk data | `OAuth` | Yes | Unknown | [Go!](https://vk.com/dev/sites) |
+| Whatsapp | Send Whatsapp messages | No | Yes | Yes | [Go!](https://faq.whatsapp.com/en/android/26000030/) |
```
Comment (davemachado) [inline README.md]: This is not a stateless API; Rather, it is a unique URL generator for users of Whatsapp. Unfortunately this wont be appropriate for our collection here.

File: README.md (hunk 1 above)
Comment (fayazara) [reply README.md]: They do not have a Official documentation

**Comment by davemachado** [comment]
bump - this will be closed if not updated.

**Pull request closed without merging**